src/main/java/net/neoforged/neoforge/fluids/FluidUnits.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright (c) NeoForged and contributors
+ * SPDX-License-Identifier: LGPL-2.1-only
+ */
+
+package net.neoforged.neoforge.fluids;
+
+import net.minecraft.resources.ResourceLocation;
+
+public class FluidUnits {
+    public static final FluidUnit BUCKET = c("bucket");
+    public static final FluidUnit BOTTLE = c("bottle");
+    public static final FluidUnit BOWL = c("bowl");
+
+    public static final FluidUnit BLOCK = c("block");
+    public static final FluidUnit INGOT = c("ingot");
+    public static final FluidUnit NUGGET = c("nugget");
```

Comment:
We should add some javadocs with any required standards, along with suggestions on which fluids they are used on. For instance, a nugget should be `1/9` of an ingot (or maybe we say an ingot is `9x` a nugget); if you break that standard things will break.

On the other hand, your choice of nugget/ingot/block standard is less important with this system, we could recommend some values but say its not required. e.g.

> Value of fluid that can be converted one to one with a nugget. Expected to be 1/9 of {@link #INGOT}. Typically is 10mb (1/100 of a bucket), but other standards may exist.

On `BUCKET` and `MB` the value being enforced could be mentioned. `GEM` may have more standard examples to list. `BLOCK` would be very non-standard but should at least mention other related units (such as `INGOT` or `GEM` which are typically 4 or 9)